new-admin/src/views/tools/infoclick.jsx

```diff
@@ -368,7 +371,21 @@ class ToolOptions extends Component {
               value={this.state.height}
             />
           </div>
+          <div className="separator">Generella inställningar</div>
           {this.renderVisibleForGroups()}
+          <div>
+            <input
+              id="allowDangerousHtml"
+              name="allowDangerousHtml"
+              type="checkbox"
+              onChange={(e) => {
+                this.handleInputChange(e);
+              }}
+              checked={this.state.allowDangerousHtml}
+            />
+            &nbsp;
+            <label htmlFor="allowDangerousHtml">Tillåt HTML i infoclick</label>
```

Comment:
Thought we didnt allow html at all now, only markdown?

Reply:
There's support for both formats in ReactMarkdown. I'd like to shut it off, but it would meet a lot of resistance in the admins camp, so we give them an option to keep it on.